app/main/appium-method-handler.js

```diff
@@ -64,7 +64,11 @@ export default class AppiumMethodHandler {
   }
 
   async fetchElement (strategy, selector) {
+    const start = process.hrtime.bigint();
```

Comment:
Please use the Timer class from appium-support lib

Reply:
https://github.com/appium/appium-support/blob/b90d85fde1c1ce2aee0a2511e5bbff5cc11c1549/lib/net.js#L155

Reply:
Thanks, will do